Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -237,16 +237,13 @@ homebrew-vendor-install() {
   VENDOR_URL="${!url_var}"
   VENDOR_URL2="${!url2_var}"
   VENDOR_SHA="${!sha_var}"
+  VENDOR_VERSION="$(<"$VENDOR_DIR/portable-$VENDOR_NAME-version")"
 
   if [[ -z "$VENDOR_URL" || -z "$VENDOR_SHA" ]]
   then
-    odie <<-EOS
-Cannot find a vendored version of $VENDOR_NAME for your $HOMEBREW_PROCESSOR
-processor on $HOMEBREW_PRODUCT!
-EOS
+    odie "No Homebrew $VENDOR_NAME $VENDOR_VERSION available for $HOMEBREW_PROCESSOR processors!$ruby_die"
```

Comment:
I'm not familiar with `$ruby_die`. Is it intentional?

Reply:
Whoops, leftover from a previous version where this displayed another Ruby message